db/db_test.go

```diff
@@ -20,13 +22,19 @@ func reloadDB(assert *assert.Assertions, dir string) (db *DB) {
 	return db
 }
 
+func assertDataEquals(assert *assert.Assertions, db *DB, expr string) {
+	value := db.head.Value.Data.TargetValue(db.Noms())
+	assert.True(nomdl.MustParse(db.Noms(), expr).Equals(value), "Expected %s equal %s", types.EncodeValue(value), expr)
```

Comment:
You can compare the refs without loading the value:

`assert.True(types.NewRef(nomdl.MustParse(db.Noms(), expr))).Equals(db.head.Value.Data))`

Reply:
I will do that but if it fails I still want to load it to give a nicer error message.